sdk/src/abi/ddc_bucket.json

```diff
@@ -1,9 +1,9 @@
 {
   "metadataVersion": "0.1.0",
   "source": {
-    "hash": "0x5e690c8fd199cbf15e7f27bf1b510235aaf5d2bfebc776f72b77f04e03c1ad3b",
```

Comment:
The deployment of the ddc_bucket contract does not change so this file should not change etiher.